### DISCUSSION TITLE
Cincinnati Contact info on App

### DISCUSSION THREAD

**Discussion by ATLCOTech** [code4recovery/12-step-meeting-list#1478] [Q&A]
How do we update the contact information on the app.? We would like to remove the phone and change the email address.

**Comment by SIA-WEBSUP** [comment]
When you connect your feed to the meeting guide app, you have the option of supplying the information that is displayed for your AA entity, 

Simply fill out the *Meeting Guide Connection Form* to connect a new feed or update an existing one.
    see: https://meetingguide.helpdocs.io/article/g0ykqkdq0u-connecting-to-meeting-guide-step-by-step
    
![image](https://github.com/user-attachments/assets/098ad088-3645-45ea-bfe1-4b29ef440aca)
